thresholdkey.go

```diff
@@ -8,27 +8,47 @@ import (
 	"math/big"
 )
 
+// Public key for a threshold Paillier scheme.
+//
+// `V` is a generator in  the cyclic group of squares Z_n^2 and is used to
+// execute a zero-knowledge proof of a received share decryption.
+//
+// `Vi` is an array of verification keys for each decryption server `i` used to
+// execute a zero-knowledge proof of a received share decryption.
```

Comment:
One thing that occurs to me is that it would be nice to align the terminology used in this threshold implementation with the one we're using in keep-core right now. But, obviously that would mean we would be misaligned with the paper. Thoughts on which of the two we should prefer?

Reply:
(Assuming, of course, there are misalignments. There may not be.)

Reply:
I think every time we'll want to change something here, we'll have the paper opened on the second screen. In my opinion, it's better to have names like `Vi` aligned with the paper. On the other hand, if we want to have `Keep` instead of `decryption server`... 🤔  This is nice, however, we want to merge changes in this PR to the upstream repo so I am not sure if we can do this rename here.

Reply:
I also lean toward matching the paper. That way this can remain a useful lib outside of our work on Keep.